### PR TITLE
User description for tanmai-h added

### PR DESCRIPTION
**Pull request by tanmai-h**
*First pull request  to Web Club NITK for Hacktoberfest 2k17
This request is for Issue #1

### CODE REVIEW

File: C/tanmai-h.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
```
Comment (salman-bhai) [inline C/tanmai-h.c]: Please reference this Issue as well in the Pull Request!

**Comment by salman-bhai** [comment]
ANy updates?

**Comment by tanmai-h** [comment]
I'm not able to resolve the merge conflicts

**Comment by salman-bhai** [comment]
Delete the folder and push
It'll work out!

**Comment by salman-bhai** [comment]
Inside the Hacktoberfest-2k17 folder there will be another folder. Delete that and then add, commit, push to this branch and it'll automatically update this PR

**Pull request merged**